File: SecondYear/semestrul4/MPP/PROIECT/CSharpProjectNetworking/Model/Organizator.cs
using System.Text.Json.Serialization;

namespace Model;
[Serializable]
public class Organizator : Entity<int>
{
    private string _username;
    private string _password;
    
    public Organizator(){}
    
    [JsonConstructor]
    public Organizator(string username, string password)
    {
        this._username = username;
        this._password = password;
    }

    public override string ToString()
    {
        return
            $"{nameof(_username)}: {_username}, {nameof(_password)}: {_password}, {nameof(Username)}: {Username}, {nameof(Password)}: {Password}";
    }

    public string Username
    {
        get { return _username; }
        set { _username = value; }
    }
        
    public string Password
    {
        get { return _password; }
        set { _password = value; }
    }
        
    public override bool Equals(object obj)
    {
        if (this == obj) return true;
        if (obj == null || GetType() != obj.GetType()) return false;
            
        Organizator that = (Organizator)obj;
        return Equals(_username, that._username) && Equals(_password, that._password);
    }
        
    public override int GetHashCode()
    {
        return HashCode.Combine(_username, _password);
    }
}

File: SecondYear/semestrul4/MPP/PROIECT/JavaProjectORM/ProjectModel/src/main/java/ro/mpp2025/javaprojectui/Organizator.java
package ro.mpp2025.javaprojectui;

import java.util.Objects;

public class Organizator extends Entity<Integer> {

    private String username;
    private String password;


    public Organizator(String username, String password) {
        this.username = username;
        this.password = password;
    }


    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Organizator that = (Organizator) o;
        return Objects.equals(username, that.getUsername()) && Objects.equals(password, that.getPassword());
    }

    @Override
    public int hashCode() {
        return Objects.hash(username, password);
    }

    @Override
    public String toString() {
        return "Organizator{" +
                "org_id=" + getId() +
                ", username='" + username + '\'' +
                ", password='" + password + '\'' +
                '}';
    }
}
